### ISSUE TITLE
FIND: Parameter format not correct .. then the video title keeps repeating without downloading

### ISSUE THREAD

**Issue by psychicEgg** [knowbee/llvd#115]
**Running Python 3.11.4**

Hiya, first time using your app. I've followed your instructions, grabbed the two cookie values, saved them into cookie.txt, but when I run:
```
llvd -c "artificial-intelligence-foundations-machine-learning-22345868" -ca --cookies
```
It displays an error: FIND: Parameter format not correct

Then it will download one video, and then keep repeating the next video title endlessly.

Here's a screenshot:

![llvd](https://github.com/knowbee/llvd/assets/69336657/72bb8e7a-c935-4bf6-a30f-18c384db5818)

Did I do something wrong?

Thanks for your help!

Egg.

**Comment by knowbee** [comment]
What happens when you kill it and run it again ? does it fail on the same video ?

**Comment by psychicEgg** [comment]
> What happens when you kill it and run it again ?

It's really sporadic. If I kill (CTRL C) the process then restart it will sometimes download the video on the next restart, but other times it will take several restarts. 

> does it fail on the same video ?

95% of the time it will fail on the second video if it successfully downloads the first. The other 5% it will fail on the third video after downloading the first two. I'll attach a screenshot from powershell.

I wonder if it might have something to do with logging in via my library (library card number and password). Maybe there's an extra cookie value being checked? But then I guess it wouldn't authenticate at all?

Does the "FIND: Parameter format not correct" alert mean anything important?

![image](https://github.com/knowbee/llvd/assets/69336657/901a0111-bdc8-4d29-9645-2c8beda2a275)

**Comment by knowbee** [comment]
correct." Feel free to debug it and raise a P if I don't fix this sooner; I'll make a release once I figure it out.

**Comment by knowbee** [comment]
I'll have to look further. I'm a little unfamiliar with "FIND: Parameter format not correct." Feel free to debug it and raise a P if I don't fix this sooner; I'll make a release once I figure it out.

**Comment by psychicEgg** [comment]
I'm not sure how useful I'll be as I'm using LL to learn Python :) If it's working ok on your end then I'd say it might have something to do with an extra cookie value for my local library (as in, the place with books). I have to login via the library portal (on the LL site), and then enter my library card number and password. I can't simply login via the main LL page.

This is the list of cookie keys:

![image](https://github.com/knowbee/llvd/assets/69336657/a807e4d2-75cc-48f0-9c08-5630108c64e8)

Don't worry about it if you're busy, I need to watch the videos anyway and the membership is unlimited. 

Cheers!

**Comment by luckydevil13** [comment]
same here with FIND: Parameter format not correct.
windows 10(cmd)

```
c:\Users\Andrei_Tsviatsinski\Downloads>llvd -c "effective***" --caption -r 720 --cookies
Using cookie info from cookies.txt
FIND: Parameter format not correct

Current: 0. Introduction/01. Improving ...@720p
Downloading video... : 100%|✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹✹| 33.6M/33.6M [00:04<00:00, 7.14MB/s]
Downloading subtitles..
....

Current: 1. Assessing Your Listening Skills/02. Recalling details.mp4 @720p

Current: 1. Assessing Your Listening Skills/02. Recalling details.mp4 @720p

Current: 1. Assessing Your Listening Skills/02. Recalling details.mp4 @720p

Current: 1. Assessing Your Listening Skills/02. Recalling details.mp4 @720p

Current: 1. Assessing Your Listening Skills/02. Recalling details.mp4 @720p
```

**Comment by aYip9394** [comment]
I am kind of getting the same error but doesn't have the message "FIND: Parameter format not correct"

I think LinkedIn Learning is limiting how many requests but I am not sure

**Comment by baerchi145** [comment]
I am seeing exactly the same error message, any suggestion how to solve the issue?

**Comment by volvox13** [comment]
These are two different problems.

The "FIND: Parameter format not correct" comes from app.py:
  # remove empty files
  command = 'find . -depth -type f -size 0 -exec rm {} +'
Writing "find" in the (win 11) command line gives the same message. Also comes up only with cookie auth, as the above line is in the cookies part of the code.
This message does not affect the rest of the code.
From my point of view this is not needed, since the removal of failed download is also there later in the code, and now no 0B files are left in the folders.

I just updated from 3.0.3 to 3.0.6 after getting the #104, #105 issues.

The repeated download attempts of the same video is what we see instead of the previous "There is a connection error. Please check your connectivity." error message (and exiting). Now there is no error message (so we don't know if it was ConnectionResetError or ConnectionError, I guess we do not get back to the exception handling), just retrying the download. In the browser everything is working fine. Clearing cache on both browsers or switching to different internet connection did not make any difference.
After about 1000-1300 sec lets me download one file, so you can try increasing throttling. I encountered it after downloading many videos, so it might be linked to some LinkedIn user limit.
